pkgs/tools/misc/wootility/default.nix

```
@@ -7,14 +7,14 @@
 }:
 let
   pname = "wootility";
-  version = "3.5.12";
+  version = "4.4.5";
```

This is currently the most up-to-date version.

```suggestion
  version = "4.4.9";
```